apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,34 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I shall be able to mark a book as finished', async () => {
+    await browser.get('/');
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+
+    const readingListToggle = await $('[data-testing="toggle-reading-list"]');
+    await readingListToggle.click();
+
+    await browser.wait(
+      ExpectedConditions.textToBePresentInElement(
+        $('[data-testing="reading-list-container"]'),
+        'My Reading List'
+      )
+    );
+
+    const markFinishedButton = await $('.not-finished button.mat-icon-button');
```

Comment:
Doesn't actually look like you're adding a book to the list before searching for this.

This test fails when you run it with a clean system.